chalice/awsclient.py

```diff
@@ -64,8 +64,9 @@ def get_function_configuration(self, name):
         return response
 
     def create_function(self, function_name, role_arn, zip_contents,
-                        environment_variables=None, runtime='python2.7'):
-        # type: (str, str, str, _ENV_VAR, str) -> str
+                        environment_variables=None, runtime='python2.7',
+                        tags=None):
+        # type: (str, str, str, _STR_MAP, str, Optional[Dict[str, str]]) -> str
```

Comment:
Why was ``_STR_MAP`` not used for tags type annotation? ``_STR_MAP`` is the same as ``Optional[Dict[str,str]]`` right?